apps/learner/src/lib/helpers/audio-state.svelte.ts

```diff
@@ -7,6 +7,7 @@ const AUDIO_STATE_KEY = Symbol('AudioState');
  */
 export class AudioState {
   isPlaying = $state(false);
+  isFloatingPlayerVisible = $state(false);
```

Comment:
Should we still call it `isFloatingPlayerVisible`?

Reply:
As we discussed this morning, we will use this as temporary toggle because in the future when we add the Web Audio API and other features, we can based on the learning unit id or etc.

Reply:
Hmm.. but this state is not temporary as it will be used to track whether the floating player is visible or not.

Reply:
Like if it's for the modal player or floating player, I think we can just retain the isFloatingPlayerVisible

Reply:
Yes, this should handle the floating player logic only